### PR TITLE
pkg/cfg(ticdc): enable pull based sink by default

### PR DESCRIPTION
**Pull request by hi-rustin**
### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tiflow/issues/5928

### What is changed and how it works?
Enable pull based sink by default.

#### Tests 

 - Unit test
 - Integration test

##### Will it cause performance regression or break compatibility?
No
##### Do you need to update user documentation, design documentation or monitoring documentation?
No
### Release note 

```release-note
None
```

### CODE REVIEW

**Review by hicqu — APPROVED** [review]

**Review by nongfushanquan — APPROVED** [review]

File: cdc/processor/sinkmanager/manager_test.go [hunk 1]
```diff
@@ -135,7 +150,9 @@ func TestAddTable(t *testing.T) {
 	require.True(t, ok)
 	require.NotNil(t, tableSink)
 	require.Equal(t, 0, manager.sinkProgressHeap.len(), "Not started table shout not in progress heap")
-	manager.StartTable(tableID, 1)
+	err := manager.StartTable(tableID, 1)
+	require.Nil(t, err)
```
Comment (3AceShowHand) [inline cdc/processor/sinkmanager/manager_test.go:154]: Use`require.NoError` is preferred.

Comment (hi-rustin) [reply cdc/processor/sinkmanager/manager_test.go]: Fixed.

**Review by hi-rustin — COMMENTED** [review]
Check

**Pull request merged**